string frame_id
float64 x
float64 y
int32 yaw 
float64 duration
bool precise_positioning
---
bool sucksess
